### ISSUE TITLE
Command line Fails. I upgraded to last version and since then I had this issue in spite of downgrading.

### ISSUE THREAD

**Issue by DanielPython2021** [microsoft/terminal#12483]
### Windows Terminal version

1.13.10395.0

### Windows build number

19043.1526

### Steps to reproduce

If I open a new instance in  a new window, that instance is according to the configuration (cmd in my case), with the proper font size, etc. If I run from python a command line opening multiple cmd tabs each one with command line, title, color, ... (this was working) it now opens all the tabs, with all configurations (profiles, colors, etc.) BUT the font size is wrong (I use smaller case and seems to open the default font size value 12 instead of my configured 10) AND colorScheme is also wrong (perhaps Campbell instead of my configured half dark).

### Expected Behavior

see Steps to reproduce

### Actual Behavior

see Steps to reproduce

**Comment by zadjii-msft** [comment]
Could you share your [settings.json file](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#where-can-i-find-the-settings-file)?

I suspect this is just "the terminal is using `profiles.defaults` now instead of the default profile" (#12311, #11836 et al), but I want to make sure.

**Comment by DanielPython2021** [comment]
This is a new one since I tried from the configuration of window repair and
restart. Changes are only: font size, buffer size, initial folder, scheme
and behavior of closing procedure.
But what opens with the wrong configuration is the following command:
This worked in the previous version and now does not (instead I downgraded
wt). Actually I am using pyautogui that opens on by one the tabs (shown in
the command below), writes the title, configures the color (mimiking human
typing).
windows_realtime_cmd = 'powershell.exe wt -M --% nt -d t:/pyprogramas -p
cmd --title motion --tabColor #429ef5 python Motion_Rev7_no_append.py ; nt
-d t:/pyprogramas -p cmd --title trend_ema --tabColor #41ffff python
trend_ema_no_append_shared_mem.py ; nt -d t:/pyprogramas -p cmd --title
monitor --tabColor #8aff41 python 0_system_monitor_rev01.py ; nt -d
t:/pyprogramas -p cmd --title findeldia --tabColor #dd00ff python
findeldia.py ; nt -d t:/pyprogramas -p cmd --title o60ma --tabColor #ed5b1c
python equity_signals_corridas_o60ma_shared_mem.py ; nt -d t:/pyprogramas
-p cmd --title over21cnt --tabColor #1cedd8 python
equity_signals_corridas_ver12_shared_mem.py ; nt -d t:/pyprogramas -p cmd
--title ancho_cnt --tabColor #e8ab10 python
equity_signals_corridas_ver1_shared_mem.py ; nt -d t:/pyprogramas -p cmd
--title ancho --tabColor #cf4dff python
equity_signals_corridas_ver8_shared_mem.py ; nt -d t:/pyprogramas -p cmd
--title pend_9 --tabColor #d145be python
equity_signals_corridas_ver4_shared_mem.py ; nt -d t:/pyprogramas -p cmd
--title pend_30 --tabColor #871c4a python
equity_signals_corridas_ver6_shared_mem.py ; nt -d t:/pyprogramas -p cmd
--title high_pVW --tabColor #6434c8 python
equity_signals_corridas_ver2_shared_mem.py ; nt -d t:/pyprogramas -p cmd
--title ver6_o60ma_acum --tabColor #23d9c0 python
equity_signals_corridas_ver6_o60ma_ac_shared_mem.py ; nt -d t:/pyprogramas
-p cmd --title watchdog_001 --tabColor #7ed923 python watchdog_001.py'

**Comment by DanielPython2021** [comment]
I thought I had sent the json file but I do not see it in my answers. So
here it is. I have also changed the font size in the wt cmd and tried again
with power shell and it opens with the same previous size font (smaller
than 24). I think it is 12 instead of my configured font size of 10.
Also I forgot to comment that the first time I ran wt after upgrading it
could not find Cascadia Mono so it replaced it. But that was the only time
it did and now I think it found Cascadia Mono.

regards,
Daniel
